blog.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Binabox blog page</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <a href="/">
+        <img src="image/logo.svg" alt="Logo" />
+        </a>
+      <nav>
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">Blog</a></li>
+          <li><a href="#">Catalog</a></li>
+          <li><a href="#">Contact Us</a></li>
+        </ul>
+      </nav>
+      <img src="image/search.svg" alt="Search" />
+      <input type="search" placeholder="Search" />
+      <a href="#" target="_blank">Discord</a>
+      <img src="image/discord.svg" alt="Discort" />
+      <button>Connect</button>
+      <img src="image/wallet.svg" alt="wallet" />
+    </header>
+    <main>
+        <h1>Our blog</h1>
+        <ul>
+            <li>Home</li>
+            <li>Our Blog</li>
+            <li>The Basilisks NFT</li>
+        </ul>
+        <section>
+            <h2>Categories</h2>
+            <ul>
+                <li class="item1">Our blog(12)</li>
+                <li class="item2">Metaverse (24)</li>
+                <li class="item3">Gaming (24)</li>
+                <li class="item4">Art (24)</li>
+                <li class="item5">Robot (24)</li>
+                <li class="item6">Lauchpad (24)</li>
+                <li class="item7">Crypto (430)</li>
+            </ul>
+            <h3>Tag</h3>
```

Comment:
It looks like another one section, as for Categories
`<section>
      <h2>Tag</h2>
          <ul>
              <li></li>
             ..............
          </ul>
`

Reply:
Fixed

Reply:
Fixed 👍